test/new/smartypants_code.md

```diff
@@ -0,0 +1,9 @@
+---
+smartypants: true
+description: SmartyPants does not modify characters within <pre>, <code>, <kbd>, or <script> tag blocks.
+spec: https://daringfireball.net/projects/smartypants/
+---
+<pre>&amp;</pre>
+<code>--foo</code>
+<kbd>---foo</kbd>
+<script>--foo</script>
```

Comment:
I was pleasantly surprised by the form the test suite takes: adding a specification was _so easy_, good work Team Marked.

😍